data/faq.yaml

```
@@ -6,4 +6,9 @@
     We are still working on the first production-ready release of OpenTelemetry. For those who want to start instrumenting production code immediately, use either [OpenCensus](https://opencensus.io) or [OpenTracing](https://opentracing.io). OpenTelemetry will provide compatibility bridges and there is no need to wait for production-ready OpenTelemetry APIs in your language.
 - q: I want to help influence the future direction of cloud-native telemetry. What should I do?
   a: |
-    You should join us in the OpenTelemetry community! You can find more details in the [OpenTelemetry community repo](https://github.com/open-telemetry/community) on GitHub: all are welcome and there’s lots to do.
+    Excellent! The [OpenTelemetry community repo](https://github.com/open-telemetry/community) has the full details, but here a re a few good starting points:
+    - Join our [community mailing list](https://lists.cncf.io/g/cncf-opentelemetry-community)
+    - Keep up to date with the [OpenTelemetry blog](https://medium.com/opentelemetry)
+    - We also have a very active chatroom on [Gitter](https://gitter.im/open-telemetry/community)
+    - All project meetings are shared on the public calendar ([web](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com&ctz=America%2FLos_Angeles), [gCal](https://calendar.google.com/calendar?cid=Z29vZ2xlLmNvbV9iNzllM2U5MGo3YmJzYTJuMnA1YW41bGY2MEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t), [iCal](https://calendar.google.com/calendar/ical/google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com/public/basic.ics)), and we recommend attending the monthly community meetings
```

```suggestion
    - All project meetings are shared on the public calendar ([web](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com&ctz=America%2FLos_Angeles), [gCal](https://calendar.google.com/calendar?cid=Z29vZ2xlLmNvbV9iNzllM2U5MGo3YmJzYTJuMnA1YW41bGY2MEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t), [iCal](https://calendar.google.com/calendar/ical/google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com/public/basic.ics)), and we recommend attending the monthly community meeting
```

will singular be better?

---

Yep